Gemfile

```diff
@@ -2,7 +2,7 @@ source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 gemspec
 
 group :extra do
-  gem 'rugged', '>= 0.24.0', '< 0.24.6', :platforms => :ruby
+  gem 'rugged', '~> 1.0.0', :platforms => :ruby
```

Comment:
If this works, would we want to loosen it to `~>1.0` so we get new Y versions?

Reply:
Up to maintainers... 

I think we want the rugged change in a standalone PR too.

Reply:
Separate PR is up that fully documents the Rugged dep bump - https://github.com/puppetlabs/r10k/pull/1064